tests/synth_test.py

```diff
@@ -623,8 +623,12 @@ def tearDown(self):
         self.f_triv = None
 
     def test_gr1c_basic(self):
-        assert isinstance(synth.synthesize("gr1c", self.f_triv),
-                          transys.MealyMachine)
+        g = synth.synthesize("gr1c", self.f_triv)
+        assert isinstance(g, transys.MealyMachine)
 
     def test_unrealizable(self):
         assert synth.synthesize("gr1c", self.trivial_unreachable) is None
+
+
+if __name__ == '__main__':
+    multiple_env_actions_test()
```

Comment:
In tests I use the `__main__` stanza for running individual tests during debugging, so the test `multiple_env_actions_test` must have been the subject of debugging. Otherwise, I just call `run_tests.py` (in particular I have placed an older version in my `$PATH` to use it in other projects too).